whylog/tests/tests_rulesbase/__init__.py

```diff
@@ -0,0 +1,36 @@
+from unittest import TestCase
+from whylog.rulesbase import YamlConfigAccess
+
+class TestYamlConfigAccess(TestCase):
+    def test_should_return_parsers_with_regex(self):
+        #given
+        parsers_path = "whylog/tests/tests_rulesbase/test_files/simple_parsers.yaml" 
```

Comment:
this will not work on Windows. Please use os.path.join() rather than manually insert the slashes into strings to construct paths from them